src/Desktop/ColorContrastAnalyzer/ColorContrastConfig.cs

```diff
@@ -1,44 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-namespace Axe.Windows.Desktop.ColorContrastAnalyzer
-{
-    public static class ColorContrastConfig
-    {
-        /**
-         * The maximum width we expect Text to be. This number can be much larger than this
-         * and still be helpful.
-         */
-        public static readonly int MaxTextThickness = 20;
-
-        /**
-         * The number of Color transitions we expect to make us believe a "Row" of Pixels
-         * contains Text.
-         */
-        public static readonly int MinNumberColorTransitions = 4;
-
-        /**
-         * We will inevitably find other color combinations in a row. To be highly confident
-         * we have text, we expect the most likely color combination to be at least this dominant.
-         * Type: Double between 0 and 1.
-         */
-        public static readonly double TextColorPairDominanceValue = .2;
```

Comment:
The `TextColorPairDominanceValue` field is not used anywhere in the code, so I went ahead and removed it from the interface